KerbalStuff/blueprints/api.py

```diff
@@ -617,9 +614,9 @@ def create_list() -> Union[Dict[str, Any], Tuple[Dict[str, Any], int]]:
 @api.route('/api/mod/create', methods=['POST'])
 @json_output
 @user_required
-def create_mod() -> Union[Dict[str, Any], Tuple[Dict[str, Any], int]]:
+def create_mod() -> Tuple[Dict[str, Any], int]:
     if not current_user.public:
-        return {'error': True, 'reason': 'Only users with public profiles may create mods.'}, 403
+        return {'error': 'Only users with public profiles may create mods.'}, 403
```

Comment:
Adding the following overwrite seems to do it. We'd loose dropbox' built-in error-overlay, but I don't think that's a issue.
```
    error: (file, errorMessage, xhr) ->
        alert = $("#error-alert")
        alert.textContent = errorMessage.reason
        alert.classList.remove('hidden')
```

Reply:
Brilliant! Force push for this coming right up...

Reply:
Ugh, I've given you only halfway jQueryfied code there, which doesn't work, sorry.
I've pushed a commit with this corrected, together with a fix to restore the automatic redirect to the edit page from #293. I've decided to do append the `?new=true` server-side instead of in the JS code, which I think is cleaner (and easier to find if someone's trying to find out where this happens).